homeassistant/components/tado/strings.json

```
@@ -54,6 +54,15 @@
       }
     },
     "sensor": {
+      "air_comfort_humidity": {
+        "name": "Air Comfort Humidity"
+      },
+      "air_comfort_temperature": {
+        "name": "Air Comfort Temperature"
```

Is Air Comfort a brand name? If so:
```suggestion
        "name": "Air Comfort humidity"
      },
      "air_comfort_temperature": {
        "name": "Air Comfort temperature"
```
else:

```suggestion
        "name": "Air comfort humidity"
      },
      "air_comfort_temperature": {
        "name": "Air comfort temperature"
```